eo-maven-plugin/src/main/resources/org/eolang/maven/pre/data.xsl

```diff
@@ -79,9 +79,16 @@ SOFTWARE.
               <xsl:text>}</xsl:text>
             </xsl:when>
             <xsl:when test="@data='string'">
-              <xsl:text>"</xsl:text>
-              <xsl:value-of select="text()"/>
-              <xsl:text>"</xsl:text>
+              <xsl:text>new StringBuilder()</xsl:text>
```

Comment:
@andreoss what is the reason for this? I don't understand why can't we just print the string as is. Why do you break it into pieces and then assemble back?

Reply:
@yegor256 Otherwise it will be compiled to a code like that which won't compile
```
"first line
second line"
```
Another solution would be to handle escape sequences https://github.com/cqfn/eo/issues/480 directly (i.e by escaping new lines in XML and converting them to "\n" in Java)

Reply:
@andreoss let's fix #480 first

Reply:
@yegor256 Done, now it uses the same syntax as strings

Reply:
@yegor256 Also, I kept `StringBuilder`.
There is no way to print a multi-line string as is, because there is no text blocks in jdk 8. Also, it seems that after https://github.com/cqfn/eo/issues/348, we wouldn't be able to print strings directly to java code, and use `StringBuilder` as well

Reply:
@andreoss I still don't understand. If you don't use StringBuilder, the text will look like `"Hello, \n world!"`, right? I mean, if you are trying to escape some symbols that may break Java syntax, then why you only care about `\n`? What about `\r`?